### PR TITLE
Remove unused Kusama Snowbridge code

### PR DESCRIPTION
**Pull request by acatangiu**
Kusama does not have Ethereum Snowbridge deployed. This commit removes all that unused code.

If we decide to deploy Kusama<>Ethereum bridge, we can bring it back.

### CODE REVIEW

File: system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs [hunk 1]
```diff
@@ -554,13 +545,7 @@ construct_runtime!(
 		BridgePolkadotMessages: pallet_bridge_messages::<Instance1> = 53,
 		XcmOverBridgeHubPolkadot: pallet_xcm_bridge_hub::<Instance1> = 54,
 
-		// Ethereum bridge pallets.
-		EthereumInboundQueue: snowbridge_pallet_inbound_queue = 80,
-		EthereumOutboundQueue: snowbridge_pallet_outbound_queue = 81,
-		EthereumBeaconClient: snowbridge_pallet_ethereum_client = 82,
-		EthereumSystem: snowbridge_pallet_system = 83,
```
Comment (acatangiu) [inline system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs:561]: These were never used by anyone so I don't think we need to save their indices, we can safely reuse index 80-83

**Review by bkchr — APPROVED** [review]

**Review by bkontur — APPROVED** [review]
and also we can remove dependencies: https://github.com/acatangiu/runtimes/pull/1

**Review by franciscoaguirre — APPROVED** [review]

**Review by joepetrowski — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Bounties: Remove payout delay ([polkadot-fellows/runtimes#386](https://github.com/polkadot-fellows/runtimes/pull/386)).
 - Polkadot System Chains: Reduce the base transaction fee by half ([polkadot-fellows/runtimes#398](https://github.com/polkadot-fellows/runtimes/pull/398)).
 - Asset Hubs: setup auto incremented asset id to 50_000_000 for trust backed assets ([polkadot-fellows/runtimes#414](https://github.com/polkadot-fellows/runtimes/pull/414)).
+- Kusama: Remove unused Snowbridge code and configs ([polkadot-fellows/runtimes#411](https://github.com/polkadot-fellows/runtimes/pull/411).
```
Comment (ggwpez) [inline CHANGELOG.md:56]: There is also a `Removed` section 🙈

**Review by ggwpez — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (bkchr) [reply CHANGELOG.md:56]: @acatangiu please move the line and then we can merge this.

Comment (acatangiu) [reply CHANGELOG.md]: done

**Pull request merged**